cloud

```diff
@@ -965,22 +986,25 @@ function execute_command() {
             --smtp-password-secret="$smtp_password_secret" \
             --database="$database"
         sections_run "$sections" scheduler_deploy \
-            "$project" "$prefix" "$trigger_topic"
+            "$project" "$prefix" "$load_queue_trigger_topic" \
+             "$pick_notifications_trigger_topic"
         sections_run "$sections" submitters_deploy \
             "$project" "$new_topic" "${submitters[@]}"
     elif [ "$command" == "withdraw" ]; then
         sections_run "$sections" submitters_withdraw \
             "$project" "$new_topic" "${submitters[@]}"
         sections_run "$sections" scheduler_withdraw \
-            "$project" "$prefix" "$trigger_topic"
+            "$project" "$prefix" "$load_queue_trigger_topic"
         sections_run "$sections" cloud_functions_withdraw "$project" "$prefix"
         sections_run "$sections" firestore_withdraw \
             "$project" "$spool_collection_path"
         sections_run "$sections" secrets_withdraw \
             "$project" "$smtp_password_secret"
         sections_run "$sections" pubsub_withdraw \
             --project="$project" \
-            --trigger-topic="$trigger_topic" \
+            --load-queue-trigger-topic="$load_queue_trigger_topic" \
+            --pick-notifications-trigger-topic= \
```

Comment:
Please remove the "equals" character (`=`) from the option's end. It's not generally supposed to be there, if its value is in a separate argument.